src/main/java/com/zendesk/maxwell/schema/ddl/MysqlParserListener.java

```diff
@@ -36,6 +38,8 @@ public MaxwellSQLSyntaxError(String message) {
 
 	private final LinkedList<ColumnDef> columnDefs = new LinkedList<>();
 
+	private ArrayList<String> pkColumns;
```

Comment:
can't we do  `= new ArrayList<>();` to avoid the ugly check above? (just nitpic)